middleware/i18n/i18n.go

```diff
@@ -14,27 +14,32 @@ import (
 	"github.com/pkg/errors"
 )
 
-// LanguageFinder can be implemented for custom finding of search
+// LanguageFinder - deprecated
```

Comment:
what should they use instead?

Reply:
I just added more infos in the last commit, thanks!